nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1394,30 +1394,6 @@ static void nxagentParseSingleOption(char *name, char *value)
 
     return;
   }
-  else if  (strcmp(name, "clipboard") == 0)
-  {
-    if ((strcmp(value, "both") == 0) || (strcmp(value, "1") == 0))
-    {
-      nxagentChangeOption(Clipboard, ClipboardBoth);
-    }
-    else if (strcmp(value, "client") == 0)
-    {
-      nxagentChangeOption(Clipboard, ClipboardClient);
-    }
-    else if (strcmp(value, "server") == 0)
-    {
-      nxagentChangeOption(Clipboard, ClipboardServer);
-    }
-    else if ((strcmp(value, "none") == 0) || (strcmp(value, "0") == 0))
-    {
-      nxagentChangeOption(Clipboard, ClipboardNone);
-    }
-    else
-    {
-      nxagentChangeOption(Clipboard, ClipboardBoth);
-    }
-    return;
-  }
```

Comment:
Removing this is not a good idea, I guess. It removes functionality for setting clipboard security via nx/nx options. Please fix this.

Reply:
No, it does not, because the identical code is found elsewhere, too. This here is never reached